internal/run/minimumViableSpec.go

```diff
@@ -26,12 +26,19 @@ func (w *Workflow) retryWithMinimumViableSpec(ctx context.Context, parentStep *w
 	substep := parentStep.NewSubstep("Retrying with minimum viable document")
 	source := w.workflow.Sources[sourceID]
 
-	source.Transformations = append(source.Transformations, workflow.Transformation{
-		FilterOperations: &workflow.FilterOperationsOptions{
-			Operations: strings.Join(invalidOperations, ","),
-			Exclude:    pointer.ToBool(true),
-		},
-	})
+	if len(invalidOperations) > 0 {
```

Comment:
just checking we only save this source if it works right?

Reply:
Correct